github/code-scanning.go

```diff
@@ -24,6 +24,7 @@ type Rule struct {
 	Severity        *string  `json:"severity,omitempty"`
 	Description     *string  `json:"description,omitempty"`
 	Name            *string  `json:"name,omitempty"`
+	SeverityLevel   *string  `json:"security_severity_level,omitempty"`
```

Comment:
For consistency, can we please change the name?

```suggestion
	SecuritySeverityLevel   *string  `json:"security_severity_level,omitempty"`
```